src/vm/builtins/builtin_runner/builtin_runner.zig

```diff
@@ -47,8 +47,29 @@ pub const POSEIDON_BUILTIN_NAME = "poseidon_builtin";
 /// The name of the segment arena builtin.
 pub const SEGMENT_ARENA_BUILTIN_NAME = "segment_arena_builtin";
 
+pub const BuiltinName = enum {
```

Comment:
We already have this enum defined in `programjson.zig` file no?

Reply:
we need same as builtinrunner based